08-alouette/alouette_test.rb

```diff
@@ -0,0 +1,117 @@
+gem 'minitest', '>= 5.0.0'
+require 'minitest/spec'
+require 'minitest/reporters'
+require 'minitest/autorun'
+require_relative 'alouette'
+
+Minitest::Reporters.use!
+
+class AlouetteTest < Minitest::Test
+  describe "lines_for_verse" do
+    it "returns an array of strings" do
+      lines = Alouette.lines_for_verse(3)
+
+      lines.must_be_kind_of Array
+
+      lines.each do |line|
+        line.must_be_kind_of String
+      end
+    end
+
+    it "generates the correct line for the first verse" do
```

Comment:
By default, the other weekend warriors have all tests skipped except for the first one